lib/webidl2.js

```diff
@@ -82,7 +130,7 @@
     }
 
     toString() {
-      return `${this.message}, line ${this.line} (tokens: '${this.input}')\n${JSON.stringify(this.tokens, null, 4)}`;
+      return `${this.message}, line ${this.line} (tokens: ${JSON.stringify(this.input)})\n${JSON.stringify(this.tokens, null, 4)}`;
```

Comment:
nit: line length.

Reply:
I'm not sure what "line length" mean here... This is done to escape linebreaks:

```js
console.log(JSON.stringify("abc\nbcd"));
// "abc\nbcd" instead of:
// abc
// bcd

Reply:
Maybe better to do `.replace()` rather than JSON.stringify..

Reply:
sorry, I meant that this line of code is getting very long :) 
Could be: 

```JS
const input = JSON.stringify(this.input);
const tokens = JSON.stringify(this.tokens, null, 4);
return `${this.message}, line ${this.line} (tokens: ${input})\n${token}`;
```

Reply:
Ah, okay, thanks!